From bf2dead93151ed6e7f5145fd50b61c06f63e4ef3 Mon Sep 17 00:00:00 2001
From: Hannes Frederic Sowa <hannes@redhat.com>
Date: Wed, 18 Jun 2014 16:05:05 -0400
Subject: [redhat] configs: enable RANDOM32_SELFTEST in debug builds

Message-id: <e39e6abd6397915051f929a43cc514972004af27.1403107398.git.hannes@redhat.com>
Patchwork-id: 84260
O-Subject: [PATCH RHEL7.1 net] enable RANDOM32_SELFTEST in debug builds
Bugzilla: 1110197
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: Daniel Borkmann <dborkman@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110197
Build Info: https://brewweb.devel.redhat.com/taskinfo?taskID=7591387
Tested: verified that selftests runs in debug kernel; not in non-debug

Enable RANDOM32_SELFTEST in rhel7 debug builds. The selftests execute
very quickly thus we always should enable them in debug builds.

Signed-off-by: Hannes Frederic Sowa <hannes@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/debug/CONFIG_RANDOM32_SELFTEST b/redhat/configs/debug/CONFIG_RANDOM32_SELFTEST
new file mode 100644
index 0000000..dc76d5c
--- /dev/null
+++ b/redhat/configs/debug/CONFIG_RANDOM32_SELFTEST
@@ -0,0 +1 @@
+CONFIG_RANDOM32_SELFTEST=y
-- 
1.7.1